在docker环境下无法安装！使用官方dockerhub的Dockerfile 也不可以什么情况？？

ERROR! autoconf required! install autoconf first

Makefile:4: build_config.mk: No such file or directory
make: *** No rule to make target `build_config.mk'.  Stop.
在docker下 总是报这个错误，使用docker hub上的Dockerfile 也不行，但是使用虚拟机环境就可以！
而且 autoconf 已经安装过了！直接使用下载来的镜像可以运行！但是自己构建 就报错！
怎么解决？？？

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,10 +5,10 @@ MAINTAINER wendal "wendal1985@gmail.com"
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && \
-  apt-get install -y --force-yes git make gcc g++ && apt-get clean && \
+  apt-get install -y --force-yes git make gcc g++ autoconf && apt-get clean && \
   git clone --depth 1 https://github.com/ideawu/ssdb.git ssdb && \
   cd ssdb && make && make install && cp ssdb-server /usr/bin && \
-  apt-get remove -y --force-yes git make gcc g++ && \
+  apt-get remove -y --force-yes git make gcc g++ autoconf && \
   apt-get autoremove -y && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
   cp ssdb.conf /etc && cd .. && yes | rm -r ssdb

